src/mod_auth_cas.c

```diff
@@ -22,7 +22,9 @@
  *
  */
 
+#ifdev HAVE_ERROR_H
```

Comment:
Of course !
Sorry, I'm not sure how it happened, i don't have it in my original "patch" :(